content/Community/contact.en.md

```diff
@@ -8,27 +8,37 @@ author: "Johan Ljunggren"
 
 #### #DSC channel
 
-The community gathers in the PowerShell #DSC channel which is an open
-forum for all DSC related questions, discussions and collaboration.
+The community gathers in the Virtual PowerShell User Group #DSC channel
+which is an open forum for all DSC related questions, discussions and
```

Comment:
I'd argue it's not a forum in the electronic sense, but a chat or instant messaging chat room... Forum is what's on the PowerShell.org site.

Reply:
Done. Agree with this.